packages/smooth_app/lib/data_models/up_to_date_changes.dart

```diff
@@ -1,3 +1,5 @@
+// ignore_for_file: deprecated_member_use
```

Comment:
What are we ignoring here, please, at least add a todo to handle this later

Reply:
Thank you @M123-dev for your quick review!

I'll replace `ignore_for_file` with line-level `ignore`.
Most important: as I start working on #3430 I'll probably (hopefully) fix that.